sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v2/metadata.yaml

```diff
@@ -12,5 +12,5 @@ scheduling:
 bigquery:
   time_partitioning:
     type: day
-    field: creation_date
+    field: submission_date
```

Comment:
The table will need to be recreated for this to take effect.

Reply:
yes, needs both